snippet/__tests__/pipe.test.js

```diff
@@ -0,0 +1,17 @@
+const pipe = require("../index.js").pipe;
+
+describe("pipe()", () => {
+  const functionSpy1 = sinon.spy();
+  const functionSpy2 = sinon.spy();
+  const result = pipe(functionSpy1, functionSpy2);
+
+  it("returns a function", () => {
+    expect(typeof result).toBe("function");
+  });
+
+  it("all provided functions are called once when result function is called", () => {
```

Comment:
try using present tense again :)

`it("invokes all functions passed in")...`...